webpack.config.js

```diff
@@ -9,6 +9,13 @@ const Mode = require("frontmatter-markdown-loader/mode");
 const IS_DEV = process.env.NODE_ENV === "development";
 const IS_PROD = !IS_DEV;
 
+const GA_MEASUREMENT_ID = "G-13GYPPVD4Y";
+const GA_CONFIG_OPTION = {
+  allow_google_signals: false,
+  allow_ad_personalization_signals: false,
+  debug_mode: IS_DEV,
+};
```

Comment:
will u be importing from config later? (after the other pr)

Reply:
yeah